client/src/components/menubar/index.js

```diff
@@ -246,7 +248,7 @@ class MenuBar extends React.PureComponent {
           zIndex: 3,
         }}
       >
-        <AuthButtons auth={auth} />
+        <AuthButtons auth={auth} userinfo={userinfo} />
```

Comment:
If you want to, you can do:
```suggestion
        <AuthButtons {...{auth, userinfo}} />
```

Reply:
We've avoided doing any spreading at the component level. Also, this would pass in more properties (is not equiv)

Reply:
In this case what additional properties are being assigned to props?

I thought
```javascript
props = { auth: auth, userinfo: userinfo }
```
was equal to 
```javascript
props = { ...{auth, userinfo} }
```